Integrations/Akamai_WAF/Akamai_WAF.yml

```diff
@@ -0,0 +1,270 @@
+category: Network Security
+commonfields:
+  id: Akamai WAF
+  version: -1
+configuration:
+- defaultvalue: https://example.net
+  display: Server URL (e.g., https://example.net)
+  name: host
+  required: true
+  type: 0
+- display: Client token
```

Comment:
need to add to the detailed instructions how to retrieve these

Reply:
I wanted to add instructions in the readme file, isn't it sufficient?

Reply:
better to have it also in the detailed instructions, or at least refer to the doc